### DISCUSSION TITLE
Amazing handling of configuration breaking changes

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [nvim-lualine/lualine.nvim#360] [General]
Just wanted to say that you handled the configuration breaking changes very beautifully. Cheers on a job excellently done! Have used a lot of programs and can't remember anybody doing it this gracefully. I would have known about it even if I'd not been keeping a tab on the repo and I love that it was so.

**Comment by shadmansaleh** [comment]
Thanks.
